woocommerce-services.php

```diff
@@ -249,6 +256,18 @@ class WC_Connect_Loader {
 
 		public static function plugin_deactivation() {
 			wp_clear_scheduled_hook( 'wc_connect_fetch_service_schemas' );
+
+			/**
+			 * When we deactivate the plugin after wcshipping_migration_state has started,
+			 * that means the migration is done. We can mark it as completed before we deactivate the plugin.
+			 */
+			$migration_state = WC_Connect_Options::get_option( 'wcshipping_migration_state' );
+			if ( ! class_exists(WC_Connect_API_Constants::class) ) {
+				require_once __DIR__ . '/classes/class-wc-connect-api-constants.php';
```

Comment:
When deactivating, somehow the files aren't loaded. I have to manually load them here. This is also the reason why I moved the constants into its own top layer file.